modules/subchannel/doc/content/modules/subchannel/v&v/EBR-II.md

```diff
@@ -0,0 +1,165 @@
+
+# EBR-II, SHRT-17 Validation
+
+## Test Description
+
+&nbsp;
+
+ On April 3, 1986, two tests were carried out to demonstrate the effectiveness of passive feedback in the EBR-II reactor. Both tests began from full reactor power and were initiated when both the primary coolant pumps and the intermediate loop pump were simultaneously tripped, to simulate a loss-of-flow accident. SHRT-17 was a protected loss-of-flow (LOF) test and SHRT-45R was an unprotected loss-of-flow (ULOF) test. At the beginning of test SHRT-17, the primary pumps were tripped at the same time as a full control rod insertion. SHRT-45R, was similar to SHRT-17 except that during this test the plant protection system (PPS) was disabled, to prevent it from initiating a control rod scram. In the second test, reactor power decreased due to reactivity feedback effects alone.
+
+ The EBR-II reactor vessel grid plenum sub-assembly accommodated 637 hexagonal sub-assemblies, which were installed in one of three regions: a central core, inner blanket, or outer blanket. The central core comprised the 61 sub-assemblies in the first five rows. SCM results are compared with data measured in the XX09 instrumented sub-assembly. More particularly, the code calculations are compared against temperature profile measurements in various axial elevations and the transient temperature evolution of the peak temperature in the central subchannel.
```

Comment:
```suggestion
```

Reply:
We are already explaining the same thing in the first paragraph of the 'Instrumented sub-assemblies' section